spec/spec_helper.rb

```diff
@@ -1,8 +1,62 @@
-require 'coveralls'
-require 'simplecov'
+ENV['RACK_ENV'] = 'test'
+
+require File.join(File.dirname(__FILE__), '..', 'lib/app.rb')
+
+require 'byebug'
+require 'capybara'
+require 'capybara/rspec'
+require 'database_cleaner'
+require 'dm-rspec'
+require 'pry'
+require 'rspec'
+require 'tilt/erb'
+require 'coveralls' # Makers
+require 'simplecov' # Makers
 
 SimpleCov.formatters = [
-  SimpleCov::Formatter::HTMLFormatter,
-  Coveralls::SimpleCov::Formatter
+    SimpleCov::Formatter::HTMLFormatter,
+    Coveralls::SimpleCov::Formatter
 ]
 Coveralls.wear!
+
+def app
+  App
+end
+
+# def populate_links
+#   link = Link.create(title: 'Makers Academy', url: 'http://makersacademy.se', description: 'Whatever', user_id: 1)
+#   ['start-up', 'sweden', 'incubator'].each do |tag|
+#     new_tag = Tag.first_or_create(title: tag)
+#     link.tags << new_tag
+#     link.save
+#   end
+# end
```

Comment:
Because I thought it was neat and useful and put it here as a reminder to myself in case I wanted to do something similar with this project, but now that I think about it more, I guess it's not applicable so I will delete it.  Thks.